lib/ads/ad_manager.js

```diff
@@ -844,14 +844,16 @@ shaka.ads.AdManager = class extends shaka.util.FakeEventTarget {
     if (this.config_ && this.config_.disableDASHInterstitial) {
       return;
     }
-    if (region.schemeIdUri == 'urn:mpeg:dash:event:alternativeMPD:2022') {
+    const schemeIdUri = region.schemeIdUri;
```

Comment:
The latest spec still mentions `urn:mpeg:dash:event:alternativeMPD:2022` in one of the examples, but I assume that's probably an editing mistake. It's not mentioned anywhere else.

Reply:
yes it is a mistake